htdocs/js/ui/find_replace.js

```diff
@@ -295,6 +295,20 @@ RCloud.UI.find_replace = (function() {
             if(replace_stuff_) {
                 replace_stuff_.hide();
             }
+        },
+        reset: function() {
+
+            if(find_dialog_) {
+                active_match_ = undefined;
+                build_regex(find_input_.val());
+                highlight_all();
+            }
+
+        },
+        update_search: function() {
+            if(find_dialog_ && find_dialog_.is(':visible')) {
+                this.reset();
+            }
```

Comment:
Looks like I committed the `update_search` function erroneously. It was used by `notebook_controller` at a point, but not at the point I committed it.

It is superfluous. Please ignore it.